tests/virt_autotest/login_console_as_kvm_or_xen_role.pm

```diff
@@ -0,0 +1,66 @@
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: For openSUSE virtualization test only. login in console, install kvm/xen patterns if needed.
+# Maintainer: Julie CAO <jcao@suse.com>
+
+use strict;
+use warnings;
+use base 'virt_autotest_base';
+use testapi;
+use ipmi_backend_utils;
+use Utils::Backends qw(use_ssh_serial_console is_remote_backend);
+use utils qw(zypper_call systemctl);
+use virt_autotest::utils qw(is_kvm_host is_xen_host);
+
+sub run {
+
+    #local openQA can skip TW host install
+    if (get_var('IPMI_DO_NOT_RESTART_HOST')) {
+        select_console 'sol', await_console => 0;
+        send_key 'ret';
+    }
+
+    use_ssh_serial_console !check_screen('text-logged-in-root');
```

Comment:
this looks wrong. `use_ssh_serial_console` does not take an argument

Reply:
Good catch! I missed if or unless keywords. Thanks.